edge-util/src/Microsoft.Azure.Devices.Edge.Storage.RocksDb/disk/StorageSpaceCheckerBase.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Storage.RocksDb.Disk
+{
+    using System;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+
+    public abstract class StorageSpaceCheckerBase
+    {
+        readonly PeriodicTask storageSpaceChecker;
+        readonly object updateLock = new object();
+
+        StorageStatus diskStatus;
```

Comment:
>diskStatus [](start = 22, length = 10)

nit: rename to storageStatus. Might be easier to just do a string replace for disk with storage in these files as there are other places I see references to disk too